### DISCUSSION TITLE
Compatible with wagmi?

### DISCUSSION THREAD

**Discussion by nezhachen** [bcnmy/biconomy-client-sdk#490] [General]
I am integrating biconomy to my dapp.
My dapp uses wagmi.
Is biconomy compatible with wagmi?
how to read contract using biconomy?

**Comment by himanshugarg06** [comment]
Hey @nezhachen did you try https://docs.biconomy.io/Account/signers/eoa/wagmi?

**Reply by nezhachen** [reply]
I already use privy and biconomy.
I don't use alchemy provider.
